File: test/export/cs/08-operadores.cs
using System;
namespace PSeInt {
	class sin_titulo {
		static void Main(string[] args) {
			double a;
			double b;
			bool c;
			bool d;
			string e;
			string f;
			a = 11;
			b = 2;
			c = true;
			d = false;
			e = "Hola";
			f = "Mundo";
			Console.WriteLine(a+b);
			Console.WriteLine(a*b);
			Console.WriteLine(a/b);
			Console.WriteLine(a-b);
			Console.WriteLine(Math.Pow(a,b));
			Console.WriteLine(a%b);
			Console.WriteLine(a==b);
			Console.WriteLine(a!=b);
			Console.WriteLine(a<b);
			Console.WriteLine(a>b);
			Console.WriteLine(a<=b);
			Console.WriteLine(a>=b);
			Console.WriteLine(c && d);
			Console.WriteLine(c || d);
			Console.WriteLine(!(c && d));
			Console.WriteLine(e+" "+f);
			Console.WriteLine(e.Equals(f));
			Console.WriteLine(!e.Equals(f));
			Console.WriteLine(e.CompareTo(f)<0);
			Console.WriteLine(e.CompareTo(f)>0);
			Console.WriteLine(e.CompareTo(f)<=0);
			Console.WriteLine(e.CompareTo(f)>=0);
		}
	}
}

File: test/export/java/08-operadores.java
import java.io.*;
import java.math.*;
public class sin_titulo {
	public static void main(String args[]) {
		double a;
		double b;
		boolean c;
		boolean d;
		String e;
		String f;
		a = 11;
		b = 2;
		c = true;
		d = false;
		e = "Hola";
		f = "Mundo";
		System.out.println(a+b);
		System.out.println(a*b);
		System.out.println(a/b);
		System.out.println(a-b);
		System.out.println(Math.pow(a,b));
		System.out.println(a%b);
		System.out.println(a==b);
		System.out.println(a!=b);
		System.out.println(a<b);
		System.out.println(a>b);
		System.out.println(a<=b);
		System.out.println(a>=b);
		System.out.println(c && d);
		System.out.println(c || d);
		System.out.println(!(c && d));
		System.out.println(e+" "+f);
		System.out.println(e.equals(f));
		System.out.println(!e.equals(f));
		System.out.println(e.compareTo(f)<0);
		System.out.println(e.compareTo(f)>0);
		System.out.println(e.compareTo(f)<=0);
		System.out.println(e.compareTo(f)>=0);
	}
}
